ex02.exs

```diff
@@ -1,6 +1,44 @@
 
+defmodule CounterServer do
+ 
+ def init(_args) do
+    { :ok, 0 }
+ end
+
+ def handle_cast({ :set, value }, _state) do
+    { :noreply, value }
+ end
+
+ def handle_call({ :get }, _from, state) do
+    { :reply, state, state }
+ end
+
+end
+
 defmodule Ex02 do
 
+  @server CounterGlobal
```

Comment:
## -6

I'm being a little tough here, but the instructions asked you to use an agent for this, rather than writing your own GenServer.

## +2

Then I'm being a softie because you wrote your own GenServer